snapcraft.yaml

```diff
@@ -101,3 +101,84 @@ parts:
       snapcraftctl prime
       # ensure build-in tests are run
       cd ${SNAPCRAFT_PART_SRC} && make test TESTDIR=${SNAPCRAFT_PRIME}
+
+  opensc:
+    plugin: autotools
+    source: https://git.launchpad.net/ubuntu/+source/opensc
+    source-type: git
+    source-depth: 1
+    source-branch: applied/ubuntu/focal
+    autotools-configure-parameters:
+      - --host ${SNAPCRAFT_ARCH_TRIPLET}
+      - --prefix=/usr
+      - --sysconfdir=/etc/opensc
+      - --with-pkcs11-provider=/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/pkcs11/p11-kit-client.so
```

Comment:
I would prefer we use `p11-kit-proxy.so` and configure it to enable `p11-kit-client.so`. That will allow application to reconfigure with their own pkcs#11 module if they need without having to rebuild it.

Reply:
Interestingly, later in part `p11`, you configure p11-kit to load `p11-kit-client.so`. So that part is already done.

Reply:
For the proxy, please see my other comment.

Reply:
Good point for p11 part. Is the `/etc/pkcs11/modules/` only used by the `p11-kit-proxy.so`?
I thought it's also used by the `p11-tool` and reason I added it there.
With this config, `p11tool` also works without need to provide `--provider` argument